updatesparser/pkg/types/updateinfo.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright © 2024 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package types
+
+import (
+	"encoding/xml"
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+type UpdateInfo struct {
+	XMLName xml.Name `xml:"updates"`
+	Updates []Update `xml:"update"`
+}
+
+type Update struct {
```

Comment:
Just a nitpick, I would create a wrapper object, like `UpdateManifest`, to contain the list of `Update` (and possibly some other fields in the future, like refresh/creation date and what  not. 
This is to follow the best practice of always return a JSON object, instead of an array. Makes it more predictable/easier for consumers.